airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -2034,51 +2034,53 @@ interpolation:
-        - "{{ now_utc().strftime('%Y-%m-%d') }}"
+        - "'{{ now_utc() }}' -> '2021-09-01 00:00:00+00:00'"
+        - "'{{ now_utc().strftime('%Y-%m-%d') }}' -> '2021-09-01'"
     - title: Today (UTC)
       description: Returns the current date in UTC timezone. The output is a date object.
       arguments: {}
       return_type: Date
       examples:
-        - "{{ today_utc() }}"
+        - "'{{ today_utc() }}' -> '2021-09-01'"
+        - "'{{ today_utc().strftime('%Y/%m/%d')}}' -> '2021/09/01'"
     - title: Timestamp
       description: Converts a number or a string representing a datetime (formatted as ISO8601) to a timestamp. If the input is a number, it is converted to an int. If no timezone is specified, the string is interpreted as UTC.
       arguments:
         datetime: A string formatted as ISO8601 or an integer representing a unix timestamp
       return_type: int
       examples:
-        - "{{ timestamp(1646006400) }}"
-        - "{{ timestamp('2022-02-28') }}"
-        - "{{ timestamp('2022-02-28T00:00:00Z') }}"
-        - "{{ timestamp('2022-02-28 00:00:00Z') }}"
-        - "{{ timestamp('2022-02-28T00:00:00:-08:00') }}"
+        - "'{{ timestamp(1646006400) }}' -> 1646006400"
+        - "'{{ timestamp('2022-02-28') }}' -> 1646006400"
+        - "'{{ timestamp('2022-02-28T00:00:00Z') }}' -> 1646006400"
+        - "'{{ timestamp('2022-02-28 00:00:00Z') }}' -> 1646006400"
+        - "'{{ timestamp('2022-02-28T00:00:00-08:00') }}' -> 1646035200"
     - title: Max
       description: Returns the largest object of a iterable, or or two or more arguments.
       arguments:
         args: iterable or a sequence of two or more arguments
       return_type: Any
       examples:
-        - "{{ max(2, 3) }}"
-        - "{{ max([2, 3]) }}"
+        - "'{{ max(2, 3) }}' -> 3"
+        - "'{{ max([2, 3]) }}' -> 3"
     - title: Day Delta
       description: Returns the datetime of now() + num_days.
       arguments:
         num_days: The number of days to add to now
         format: How to format the output string
       return_type: str
       examples:
-        - "{{ day_delta(25) }}"
-        - "{{ day_delta(25, format='%Y-%m-%d') }}"
+        - "'{{ day_delta(1) }}' -> '2021-09-02T00:00:00.000000+0000'"
+        - "'{{ day_delta(-1) }}' -> '2021-08-31:00:00.000000+0000'"
+        - "'{{ day_delta(25, format='%Y-%m-%d') }}' -> '2021-09-02'"
     - title: Duration
       description: Converts an ISO8601 duratioin to datetime.timedelta.
       arguments:
         duration_string: "A string representing an ISO8601 duration. See https://www.digi.com/resources/documentation/digidocs//90001488-13/reference/r_iso_8601_duration_format.htm for more details."
       return_type: datetime.timedelta
       examples:
-        - "{{ duration('P1D') }}"
-        - "{{ duration('P6DT23H') }}"
-        - "{{ (now_utc() - duration('P1D')).strftime('%Y-%m-%dT%H:%M:%SZ') }}"
+        - "'{{ duration('P1D') }}' -> '1 day, 0:00:00'"
```

Comment:
this prints weird, but the result is a timedelta. The example of line 2083 shows how it can be used